guides/common/modules/ref_storage-guidelines.adoc

```diff
@@ -73,19 +79,23 @@ After a full synchronization task is completed, the content is moved to the `/va
 For content in RPM format, each RPM file is moved to the `/var/lib/pulp` directory after it is synchronized.
 A maximum of 5 RPM files are stored in the `/var/cache/pulp/` directory at any time.
 Up to 8 RPM content synchronization tasks can run simultaneously by default, with each using up to 1 GB of metadata.
+endif::[]
 
-ifeval::["{build}" != "foreman-deb"]
+ifndef::foreman-deb[]
```

Comment:
With the ifdefs, we could define what it is rather than what it isn't. 
Here, for example, we were trying to express that this was for both foreman, and satellite, but not Foreman with Debian.

Reply:
Fair, I switched to only declaring what build types should be rather than including all but.